### PR TITLE
chore: generate version range for deprecated versions

### PR DESCRIPTION
**Pull request by OrKoN**

### CODE REVIEW

File: versions.js [hunk 1]
```diff
@@ -42,4 +42,15 @@ const versionsPerRelease = new Map([
   ['73.0.3679.0', 'v1.12.2'],
 ]);
 
-module.exports = versionsPerRelease;
+const lastMaintainedChromiumVersion = '97.0.4692.0';
```
Comment (mathiasbynens) [inline versions.js:46]: Let’s add a comment here explaining which version this is supposed to refer to, i.e. the same major version as the current Chrome Stable per https://chromestatus.com/roadmap.

**Review by mathiasbynens — APPROVED** [review]
LGTM % suggestion

**Comment by mathiasbynens** [comment]
I think this is `chore:` rather than `feat:`, since it doesn’t add a new Puppeteer API or feature.

File: versions.js (hunk 1 above)
Comment (OrKoN) [reply versions.js:46]: Done, PTAL

**Review by mathiasbynens — APPROVED** [review]

**Pull request merged**